tests/unit/test_core/test_link.py

```diff
@@ -174,26 +174,19 @@ def test_use_tag__error(self, mock_is_tag_available):
         result = link.use_tag(Mock())
         self.assertFalse(result)
 
-    @patch('kytos.core.interface.Interface.make_tag_available')
-    @patch('kytos.core.interface.Interface.is_tag_available')
-    def test_make_tag_available__success(self, *args):
+    def test_make_tag_available__success(self):
```

Comment:
The mocks with patched side effects were hard to read I refactored the test cases making the intention clearer